packages/suite-web/e2e/tests/metadata/output-labeling.test.ts

```diff
@@ -86,6 +85,31 @@ describe('Metadata - Output labeling', () => {
             cy.getTestElement('@metadata/input').clear().type('write something that wont be saved');
             cy.getTestElement('@metadata/cancel').click({ force: true });
             cy.getTestElement(`${sentToMyselfEl}`).should('contain', 'submitted by button');
+
+            // validate that exporting transactions exports also labels
+            // note: having trouble using here due to inability to scroll to that element, so I am using force here
+            // onAccountsPage.exportDesiredTransactionType('csv');
+            cy.getTestElement('@wallet/accounts/export-transactions/dropdown').click({
+                force: true,
+            });
+            cy.getTestElement(`@wallet/accounts/export-transactions/csv`).click({
+                force: true,
+            });
+
+            cy.wait(1000);
+            cy.task('readDir', Cypress.config('downloadsFolder')).then((dir: any) => {
+                cy.task('readFile', `${Cypress.config('downloadsFolder')}/${dir[0]}`).then(
+                    (file: any) => {
+                        const expectedSubstr =
+                            '1PmVvr5DNVYJygtDT7J312qmxpa5pceu9E;submitted by button';
+                        if (!file.includes(expectedSubstr)) {
+                            // huh can't figure out proper way how to assert string retrieved by cy.task command
+                            throw new Error('label not part of export');
+                        }
```

Comment:
```suggestion
                        expect(file).to.include(expectedSubstr);
```
![Screenshot 2023-06-28 at 14 07 51](https://github.com/trezor/trezor-suite/assets/33235762/a56d4177-1641-41f5-bb44-12edb8f3366c)

Reply:
in case of wrong string it fails 
![Screenshot 2023-06-28 at 14 08 52](https://github.com/trezor/trezor-suite/assets/33235762/d962ede3-9389-460b-9414-67dad22b9735)